deltas/tokenizers/wikitext_split.py

```diff
@@ -97,4 +72,10 @@
     ("etc", r"."),
 ]
 
-wikitext_split = RegexTokenizer(LEXICON)
+LEXICON_LATIN = LEXICON.copy()
+LEXICON_LATIN.insert(-2, ('cjk', cjk))
```

Comment:
Why not insert this right after "word"?

Reply:
I can do that. My thought process was since we won't have lots of cjk in a regular latin-dominant text, we don't need to handle them before the tab_open, tab_close, etc.